0-ad-history-mod-package/simulation/templates/units/han/champion_cavalry_spearman_academy.xml

```diff
@@ -8,6 +8,13 @@
     <Requirements>
       <Techs datatype="tokens">-unlock_champion_cavalry phase_city</Techs>
     </Requirements>
+    <History>
+        The Tiger and Leopard Cavalry was an elite cavalry division belonging to the warlord Cao Cao, commanded by his cousin Cao Chu. At that time, during the late Han dynasty and the early Three Kingdoms era (the first and second centuries BC), the use of cavalry was generally declining and shifting more towards heavier armor. There were other, similar cavalry divisions to the Tiger and Leopard cavalry in the armies of other warlords as well.
```

Comment:
This paragraph isn't 100% clear, is cavalry declining for infantry? Or light cavalry giving way to heavy cavalry?

Reply:
both actually

Reply:
is it better like this?